misc/image_classification_template.yaml

```diff
@@ -0,0 +1,39 @@
+kind: Template
+apiVersion: v1
+metadata:
+  name: image-classification-template
+  annotations:
+    openshift.io/display-name: "Image Classification Job Template"
+    description: >-
+      This is a template for running the mlperf image_classification benchmark job.
+    tags: "image-classifiation,benchmark,mlperf"
+    openshift.io/provider-display-name: "MLPerf"
+    openshift.io/documentation-url: "https://github.com/mlperf/training"
+    openshift.io/support-url: "support@mlperf.org"
+    message: "tbd"
+  labels:
+    template: "image-classification-template"
+    app: "training"
+
+objects:
+  - kind: Job
```

Comment:
Why is Job required ?

Reply:
it's not, and i keep (thinking I am) deleting it, and its still there. I'll try again